OMEdit/OMEdit/OMEditGUI/Annotations/TextAnnotation.cpp

```diff
@@ -290,13 +290,37 @@ void TextAnnotation::paint(QPainter *painter, const QStyleOptionGraphicsItem *op
   }
 }
 
+/*!
+ * \brief Make square be transformed to square (X and Y axes scaled equally)
+ */
+void TextAnnotation::equalizeScale(QPainter *painter, double &horizontalUnscaling)
+{
+  horizontalUnscaling = 1.0;
+  if (painter->worldTransform().isAffine()) {
```

Comment:
Consider inverting the logic here, to remove the else and make things a bit less nested

Reply:
What do you mean? `if` branches look quite different...

Reply:
Ah it is simple a matter of taste 💯 

Some clarification in code:

```void TextAnnotation::equalizeScale(QPainter *painter, double &horizontalUnscaling)
{
  horizontalUnscaling = 1.0; // <-- should this always be set? Considering it is passed by reference it might be unwise
  if (!painter->worldTransform().isAffine()) {
    return;
  }
  const QMatrix matr = painter->worldMatrix();
  const double qx = std::max(std::abs(matr.m11() + matr.m12()), 0.01);
  const double qy = std::max(std::abs(matr.m21() + matr.m22()), 0.01);
  if (qx < qy) {
    painter->scale(1.0, qx / qy);
    return;
  } 
  painter->scale(qy / qx, 1.0);
  horizontalUnscaling = qy / qx; 
}
```